tests/step-029.sh

```diff
@@ -2,7 +2,7 @@
 # Test:
 #   Direct template execution: execute auto-update
 
-if ! curl --version && ! wget --version; then
+if ! curl --version >/dev/null 2>&1 && ! wget --version >/dev/null 2>&1; then
```

Comment:
Not sure why this needs to change? I think it's good to have their version printed when available, so it's easier to see when it goes wrong. If the test is successful, their output will be hidden anyway.

Reply:
True!